### PR TITLE
Updating assignments with the new UI

### PR DESCRIPTION
**Pull request by stephaniegiang**
## What
Taking the changes from https://github.com/education/classroom/pull/1835/ 🎉 @srinjoym was slowly updating, but going to take over.

Updating the assignment page with the new UI. Also removing custom CSS styling and using Primer instead. 

image

image

### CODE REVIEW

**Comment by stephaniegiang** [comment]
Love to get some reviews on this

**Comment by stephaniegiang** [comment]
@femmebot the UI for assignments currently looks like this :

image

This PR just switches us from using custom css elements to Primer. Also I like that copy a lot better. I'll make that change 😄

**Comment by stephaniegiang** [comment]
Made the changes for the new text copy 😄 
image

Ready for another round of reviews!

File: config/locales/views/organizations/en.yml [hunk 1]
```diff
@@ -77,5 +77,5 @@ en:
       set_of_teams: 'Set of teams'
       no_teams: "This classroom doesn't have any sets of teams yet."
       assignments: 'Assignments'
-      no_assignments: "This organization doesn't have any assignments yet."
+      no_assignments: "There are no assignments in this classroom yet."
```
Comment (thebradbain) [inline config/locales/views/organizations/en.yml]: Love the change from organizations to classrooms ✨

File: app/assets/stylesheets/layout.scss [hunk 2]
```diff
@@ -214,47 +214,6 @@ main {
   background-color: #555;
 }
 
-.assignment-list-items {
```
Comment (thebradbain) [inline app/assets/stylesheets/layout.scss]: Love that you're getting rid of custom CSS and moving to Primer 💯 🙌

**Pull request merged**